src/common/deepLinking.js

```diff
@@ -21,6 +21,27 @@ const withMetaItem = ({ metaItem }) => {
     };
 };
 
+const withMetaResource = ({ metaResource, type, id }) => {
+    const queryParams = new URLSearchParams([['metaTransportUrl', metaResource.request.base]]);
+    const videoId = metaResource.content.type === 'Ready' ?
```

Comment:
the logic about default video id doesnt apply in detail page because user may have another videoid selected and change source while he is on the streams page.
move videoId to arguments as the example in Slack